dask_expr/tests/test_distributed.py

```diff
@@ -90,6 +91,28 @@ async def test_merge_index_precedence(c, s, a, b, shuffle):
     lib.testing.assert_frame_equal(x.sort_index(ascending=False), pdf.join(pdf2))
 
 
+@gen_cluster(client=True)
+@pytest.mark.parametrize("shuffle", ["tasks", "p2p"])
+@pytest.mark.parametrize("broadcast", [True, 0.6])
```

Comment:
We will probably want to do a parameter sweep over `how` as well (since the algorithm is sometimes dependent on this).

Reply:
good point, thx